docs/pages/inboxes/debug-your-app.md

```diff
@@ -1,5 +1,107 @@
+
+- If `maybeForked` returns `true`, it is highly likely that the group chat is forked.
+- If `maybeForked` returns `false`, it is highly likely that the group chat is NOT forked.
+
+To learn about group chat forks, see [MLS Group State Forks: What, Why, How](https://cryspen.com/post/mls-fork-resolution/).
+
+If you believe you are experiencing a forked group, please [open an issue](https://github.com/xmtp/libxmtp/issues) in the LibXMTP repo to get support. Please include logs, the epoch, and other fork details.
+
+Forked groups are not recoverable. Your options are to:
+
+- Remove all members from the forked group and then re-add them to the group.
+- Start a new group.
+
+## File logging
+
+You can use file logging to debug complex issues. It keeps a 6-hour window of logs that get written and rolled every hour.
+
+## Network statistics
+
+A client has a function called `getNetworkStatistics`. These statistics are maintained per client instance, so each app installation has its own separate counters.
+
+You can use these statistics to see which and how many API, identity, and streaming calls are going across the network at any moment. This information can help you better manage network usage and debug potential rate limiting issues.
+
+### Get aggregated statistics
+
+Use the `XXXXXXXX` function to return these aggregated statistics.
+
+```text
+Aggregate Stats:
+============ Api Stats ============
+UploadKeyPackage        1
+FetchKeyPackage         2
+SendGroupMessages       5
+SendWelcomeMessages     1
+QueryGroupMessages      7
+QueryWelcomeMessages    0
+SubscribeMessages       0
+SubscribeWelcomes       0
+============ Identity ============
+PublishIdentityUpdate    1
+GetIdentityUpdatesV2     4
+GetInboxIds             2
+VerifySCWSignatures     0
+============ Stream ============
+SubscribeMessages        0
+SubscribeWelcomes       0
+```
+
+### Get an individual statistic
+
+Use the `XXXXXXXX` function to return an individual statistic as a number. For example, you can just `xxxxxx` to track `uploadKeypackage` only.
+
+### Statistic descriptions
+
+#### API statistics
+
+| Statistic | Description |
+|-----------|-------------|
+| UploadKeyPackage | Number of times key packages have been uploaded. |
+| FetchKeyPackage | Number of times key packages have been fetched. |
+| SendGroupMessages | Number of times messages have been sent to group chat and DM conversations. |
+| SendWelcomeMessages | Number of times welcome messages have been sent. |
+| QueryGroupMessages | Number of times queries have been made to fetch messages being sent to group chat and DM conversations. |
+| QueryWelcomeMessages | Number of times queries have been made to fetch welcome messages. |
+| SubscribeMessages | Number of times message subscription requests have been made. |
+| SubscribeWelcomes | Number of times welcome message subscription requests have been made. |
```

Comment:
Not sure if it's worthwhile to make these more clear for every day integrators like

subscribeWelcomes is essentially streaming conversations 
SubscribeMessages is essentially streaming messages in a conversation
etc...  

But this is good for a first pass

Reply:
Fantastic feedback -- will get this in a fast-follow!